controllers/timechaos/types.go

```diff
@@ -191,18 +191,33 @@ func (r *Reconciler) recoverPod(ctx context.Context, pod *v1.Pod, chaos *v1alpha
 		return fmt.Errorf("%s %s can't get the state of container", pod.Namespace, pod.Name)
 	}
 
-	containerID := pod.Status.ContainerStatuses[0].ContainerID
+	g := errgroup.Group{}
+	for index := range pod.Status.ContainerStatuses {
+		id := pod.Status.ContainerStatuses[index].ContainerID
```

Comment:
Can we support specifying container names in the TimeChaops definition?

Reply:
Yes. If it's needed, I will add this feature in later PR.

Reply:
It's supported now! 🚀